meta-resin-common/recipes-support/resin-persistent-logs/resin-persistent-logs/resin-persistent-logs

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+set -e
+
+. /usr/sbin/resin-vars
+
+if [ "$PERSISTENT_LOGGING" == "true" ] && [ ! -d /var/log/journal ]; then
+    systemctl start var-log-journal.mount
+    journalctl --flush
+    echo "resin-persistent-logs: Persistent logging activated."
```

Comment:
Perhaps better to use some version of "basename $0" instead of hardcoding the program name.